src/runtime/Makefile

```diff
@@ -112,6 +112,12 @@ KERNELDIR := $(PKGDATADIR)
 
 IMAGEPATH := $(PKGDATADIR)/$(IMAGENAME)
 INITRDPATH := $(PKGDATADIR)/$(INITRDNAME)
+
+ROOTFSTYPE_EXT4 := \"ext4\"
+ROOTFSTYPE_XFS := \"xfs\"
+ROOTFSTYPE_EROFS := \"erofs\"
+DEFROOTFSTYPE := $(ROOTFSTYPE_EROFS)
```

Comment:
Why is the default rootfs type changed?

Reply:
Let's use `ext4` for now until enough reasons to switch to `erofs`.
We need to show the benefits first.

Reply:
I temporarily switched back to `ext4` as the default.
We need to demonstrate the benefits first before considering switching `erofs` as the default.